src/EditorFeatures/Test/Diagnostics/DiagnosticDataTests.cs

```diff
@@ -46,7 +46,7 @@ public async Task DiagnosticData_GetText2()
             var code = @"
 ";
 
-            await VerifyTextSpanAsync(code, -1, 30, 40, 40, new TextSpan(0, code.Length));
+            await VerifyTextSpanAsync(code, 0, 30, 40, 40, new TextSpan(code.Length, 0));
```

Comment:
these tests were testing how the existing system handles bizarre/invalid spans.  The new logic has been simplified (and i think made clearer).